messages/src/message.rs

```diff
@@ -494,6 +525,12 @@ impl SpResponse {
                 // So using `size_of` is appropriate here.
                 Some(modules.selected_transceiver_count() * core::mem::size_of::<Status>())
             }
+            SpResponse::StatusV2 { modules, .. } => {
+                // NOTE: We don't `hubpack::deserialize` the `Status` objects,
```

Comment:
Copy-paste nit: `Status` should be `StatusV2` or whatever we name it.

Reply:
Deleted this comment anyways, since we *are* using `hubpack::deserialize` here